CONVENTIONS.md

```diff
@@ -0,0 +1,125 @@
+# Naming conventions
+
+## General
+
+Use `UpperCamelCase` for class names. Use `lowerCamelCase` for method names, variable names, and names of fields that are not
+both static and final. Use `UPPER_SNAKE_CASE` for names of fields that are both static and final.
+
+Method names should always be verb phrases (`tick`, `getCarversForStep`), except for builder methods and "withX" methods.
```

Comment:
"should always" -> "should generally". Just in this document, you cite 4 exceptions (builders, "withX", "fromX", "toX"), and that's forgetting fluent naming strategies like `of()`.

Reply:
Done